sui-execution/latest/sui-move-natives/src/crypto/attestation.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_binary_format::errors::PartialVMResult;
```

Comment:
let's move this function to the other PR with the Move interfaces (as they will change this code)